### ISSUE TITLE
Bootstrap images pull not working

### ISSUE THREAD

**Issue by searsandrew** [searsandrew/centreforge#8]
The new options that have been added for the 'Bootstrap' images in the Media Uploader do not work as expected.

**Comment by mread1208** [comment]
Looks like it's adding a height="1" and a width="1" on the image tag.  Is there a way to remove that?

**Comment by mread1208** [comment]
I'm able to add the pull-left pull-right classes from the media uploader and things seem to display fine for me.  What is the error exactly?

**Comment by searsandrew** [comment]
Yeah, ok, so I looked at this. They are doing what we want on the front end (we should add the BS styling to the editor...) but they are duplicating the Right & Left that are already in the Media Object.

**Comment by mread1208** [comment]
Ok, let me make sure I'm following.

When I go to add media, in the alignment drop down menu under "Attachment Display Settings", I'm presented 6 options: Left, Bootstrap Left, Center, Right, Bootstrap Right and None.  Do we need to remove "Left" and "Right" so that there are only 4 options: Bootstrap Left, Center, Bootstrap Right, and none?

**Comment by searsandrew** [comment]
Ideally it should just be Left, Center, Right, None -> but using the Bootstrap classes.

**Comment by mread1208** [comment]
Gotcha, ok.  I can take a look and see if we can't overwrite the default select options that WordPress puts in there.  I'm sure we can hook it somehow.
